config/routes.rb

```diff
@@ -295,6 +295,7 @@
   get "/:username/:slug/mod" => "moderations#article"
   get "/:username/:slug/edit" => "articles#edit"
   get "/:username/:slug/delete_confirm" => "articles#delete_confirm"
+  get "/:username/:slug/toggle_mute" => "articles#toggle_mute"
```

Comment:
Things that make DB inserts should never be get requests (except in sensible exceptions where it make sense). But this is definitely a place where it should be a `POST` or `PATCH` request.